### PR TITLE
Revision for all existing index.html, releasenotes.html, and assertio…

### PR DESCRIPTION
**Pull request by edbratt**
…n lists to use jakarta EE terminology and procedures. Moved a few files with dates in their names to remove the dates.

A few minor wording changes to jakartaee rules file (in UG).

### CODE REVIEW

File: install/connector/docs/ReleaseNotes-JCA-1.7.html [hunk 1]
```diff
@@ -110,33 +107,21 @@ <h2>JCA TCK Requirements, Facts, and Figures</h2>
...
-</table>
-
-
 <hr>
 <h2>Known Problems and Workarounds</h2>
 
 <h3>Running the JCA TCK Against Java EE RI on Windows</h3>
 <blockquote>
-<p>When configuring and running the JCA TCK against the RI on Windows, please make sure to install all related software (J2SE, the RI, the TCK, etc.) on the same drive (for example, <code>C:</code>). If you want to install on a drive other than <code>C:</code>, please be sure to change the following properties in <code>&lt;TS_HOME&gt;/bin/ts.jte:</code></p>
+<p>When configuring and running the JCA TCK against the Eclipse GlassFish Compatible Implementation (CI) on Windows, please make sure to install all related software (Java SE, the CI, the TCK, etc.) on the same drive (for example, <code>C:</code>). If you want to install on a drive other than <code>C:</code>, please be sure to change the following properties in <code>&lt;TS_HOME&gt;/bin/ts.jte:</code></p>
```
Comment (bshannon) [inline install/connector/docs/ReleaseNotes-JCA-1.7.html]: More references to "JCA" that should be fixed.

File: install/connector/docs/index.html [hunk 2]
```diff
@@ -37,15 +37,15 @@
...
-<h1>Welcome to the Java<small><sup>TM</sup></small> EE Connector Architecture Specification (JCA) Compatibility Kit, Version 1.7<br>
+<h1>Welcome to the Jakarta Connectors Technology Compatibility Kit, Version 1.7<br>
 <span class="emphasize">Your Starting Point</span></h1>
 </div>
 
 <hr>
-<h2>JCA TCK 1.7 Documentation</h2>
-<p>The Java<small><sup>TM</sup></small> EE Connector Architecture Specification (JCA) Technology Compatibility Kit (TCK) 1.7 documentation includes the following:</p>
+<h2>Jakarta Connectors TCK 1.7 Documentation</h2>
+<p>TheJakarta Connectors Technology Compatibility Kit (TCK) 1.7 documentation includes the following:</p>
```
Comment (bshannon) [inline install/connector/docs/index.html]: Missing space after "The".

File: install/connector/docs/index.html [hunk 3]
```diff
@@ -37,15 +37,15 @@
...
 <ul>
-  <li><p>The <a href="ReleaseNotes-JCA-1.7-Sept-2013.html">JCA TCK 1.7 Release Notes</a> provide late-breaking information about the JCA TCK, Version 1.7.</p></li>
+  <li><p>The <a href="ReleaseNotes-JCA-1.7.html">JCA TCK 1.7 Release Notes</a> provide late-breaking information about the JCA TCK, Version 1.7.</p></li>
```
Comment (bshannon) [inline install/connector/docs/index.html]: More "JCA".

File: install/jacc/docs/ReleaseNotes-JACC-1.5.html [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+<ul>
+ <li>EARs: 0</li>
+ <li>JARs: 41</li>
+ <li>WARs: 6</li>
+ <p/>
+ <li>Total tests:   32
+ <ul>
+    <li>Web Container:  23</li>
+    <li>EJB-lite:  9</li>
+    <li>EJB Container:  n/a (EJB Container tests are only available in full CTS 8)</li>
```
Comment (bshannon) [inline install/jacc/docs/ReleaseNotes-JACC-1.5.html]: "EJB" -> "Jakarta Enterprise Beans"

File: install/jacc/docs/index.html [hunk 5]
```diff
@@ -37,24 +37,27 @@
...
-  <li><p>The <a href="assertions/api/JACCJavaDocAssertions.html">Javadoc Assertion List</a> lists all the javadoc assertions from JACC 1.5 Specification.</p></li>
-  <li><p>The <a href="assertions/spec/JACCSpecAssertions.html">Specification Assertion List</a> lists all the specification assertions from JACC 1.5 Specification.</p></li>
+  <li>
+    <p>The <a href="ReleaseNotes-JACC-1.5.html">Jakarta Authorization TCK 1.5 Release Notes</a> provides late-breaking information about the Jakarta Authorization TCK, Version 1.5.</p></li>
+  <li><p>The <em>Jakarta Authorization 1.5 Technology Compatibility Kit User's Guide</em> provides the information you need to install, configure, and run the Jakarta Authorization TCK. The User's Guide also contains the rules you need to pass for certification. </p></li>
+  <li><p>The <a href="assertions/api/JACCJavaDocAssertions.html">Javadoc Assertion List</a> lists all the javadoc assertions from Jakarta Authorization 1.5 Specification.</p></li>
+  <li><p>The <a href="assertions/spec/JACCSpecAssertions.html">Specification Assertion List</a> lists all the specification assertions from Jakarta Authorization 1.5 Specification.</p></li>
```
Comment (bshannon) [inline install/jacc/docs/index.html]: Ideally all the file names wouldn't have "JACC" in them.

File: install/jakartaee/docs/index.html [hunk 6]
```diff
@@ -36,22 +36,24 @@
...
+<p>The Release Notes may be udpated and released through the Jakarta EE 8 Platform Specification web site:  <a href="https://jakarta.ee/specifications/full-profile/8.0/">https://jakarta.ee/specifications/full-profile/8.0/</a><a href="https://jakarta.ee/specifications/full-profile/8.0/"></a> or <a href="https://jakarta.ee/specifications/web-profile/8.0/">https://jakarta.ee/specifications/webl-profile/8.0/</a>.</p>
   </li>
   <li><p>The <em>Java Platform Enterprise Edition 8 Compatibility Test Suite User's Guide</em> provides comprehensive user documentation, including instructions for installing and configuring the test suite, and using the JavaTest test harness to execute the tests. The User's Guide also contains the rules you need to pass for certification.</p></li>
-<li><p>The <a href="./assertions/JavaEESpecAssertions.html">Specification Assertion List</a> lists all the specification assertions that are tested by the Java EE 8 CTS test suite.</p></li>
+<li><p>The <a href="./assertions/JavaEESpecAssertions.html">Specification Assertion List</a> lists all the specification assertions that are tested by the Jakarta EE 8 Platform  8 CTS test suite.</p></li>
```
Comment (bshannon) [inline install/jakartaee/docs/index.html]: The file name shouldn't have "JavaEE" in it.

File: install/jaspic/docs/assertions/JASPICJavadocAssertions.html [hunk 7]
```diff
@@ -24,7 +24,8 @@
 <body bgcolor="white">
 <br>
 <CENTER>
-<h2>Java Authentication SPI for Containers<br>JASPIC - 1.1<br>
+  <h2><!--td {border: 1px solid #ccc;}br {mso-data-placement:same-cell;}-->
+  <span data-sheets-value="{'1':2,'2':'Jakarta Authentication'}" data-sheets-userformat="{'2':513,'3':{'1':0},'12':0}">Jakarta Authentication</span> - 1.1<br>
```
Comment (bshannon) [inline install/jaspic/docs/assertions/JASPICJavadocAssertions.html]: I don't know what all this html is for but it looks like a copy/paste problem.

File: install/jaspic/docs/assertions/JASPICSpecAssertions.html [hunk 8]
```diff
@@ -24,7 +24,8 @@
 <body bgcolor="white">
 <br>
 <CENTER>
-<h2>Java Authentication SPI for Containers <br>JASPIC - 1.1<br>
+<h2><!--td {border: 1px solid #ccc;}br {mso-data-placement:same-cell;}-->
+  <span data-sheets-value="{'1':2,'2':'Jakarta Authentication'}" data-sheets-userformat="{'2':513,'3':{'1':0},'12':0}">Jakarta Authentication</span>- 1.1<br>
```
Comment (bshannon) [inline install/jaspic/docs/assertions/JASPICSpecAssertions.html]: Ditto.

File: install/jaspic/docs/index.html [hunk 9]
```diff
@@ -37,25 +37,27 @@
 </head>
 <body>
 <div align="center">
-<h1>Welcome to the Java<small><sup>TM</sup></small> Authentication SPI for Containers (JASPIC) TCK, Version 1.1<br>
+<h1>Welcome to the Jakarta<small><sup></sup></small> Authentication  TCK, Version 1.1<br>
```
Comment (bshannon) [inline install/jaspic/docs/index.html]: With the "TM" removed, the surrounding html tags can be removed as well.

File: install/jaxr/docs/JAXRJavadocAssertions.html [hunk 10]
```diff
@@ -24,7 +24,7 @@
 <body bgcolor="white">
 <br>
 <CENTER>
-<h2>Java API for XML Registries<br>JAXR 1.0 - 1.0<br>
+<h2> Jakarta Registries 1.0 - 1.0<br>
```
Comment (bshannon) [inline install/jaxr/docs/JAXRJavadocAssertions.html]: I don't know why "1.0" occurs twice.

File: install/jaxr/docs/ReleaseNotes-jaxr.html [hunk 11]
```diff
@@ -426,12 +427,12 @@ <h3>
 Developers Pack has been installed
 <p><tt>webserver.host </tt>to the host the Web server is running on
 <p><tt>webserver.port </tt>to the port the Web server is running on
-<p><tt>j2ee.home.ri </tt>to the directory where the J2EE SDK 1.3.1 is installed
+<p><tt>j2ee.home.ri </tt>to the directory where the Jakarta EE 8Compatible Implementation (e.g. Eclipse GlassFish 5.1) is installed
```
Comment (bshannon) [inline install/jaxr/docs/ReleaseNotes-jaxr.html]: Missing space after "8".

File: install/jaxrs/docs/JAXRSTCK2.1-ReleaseNotes.html [hunk 12]
```diff
@@ -36,35 +39,36 @@
...
-<span class="emphasize">Release Notes, September 2017</span></h1>
+<h1>Jakarta RESTful Web Services  Technology Compatibility Kit, Version 2.1<br>
+<span class="emphasize">Release Notes, September 2019</span></h1>
 </div>
 <hr size="3">
 
-<H2>Release Notes Update For Jersey 2.26 RI Release September 2017 </H2>
-<P> A couple of changes made in Jersey 2.26 RI prompts this note.
+<H2>Release Notes Update For Eclipse Jersey 2.26 CI Release September 2017 </H2>
+<P> A couple of changes made in Eclipse Jersey 2.26 RI prompts this note.
```
Comment (bshannon) [inline install/jaxrs/docs/JAXRSTCK2.1-ReleaseNotes.html]: "RI" -> "CI"

File: install/jaxws/docs/assertions/JAXWS23JavadocAssertions.html [hunk 13]
```diff
@@ -24,7 +24,9 @@
 <body bgcolor="white">
 <br>
 <CENTER>
-<h2>Java(TM) API for XML Web Services<br>JAXWS - 2.3<br>
+  <h2><br> 
+  <!--td {border: 1px solid #ccc;}br {mso-data-placement:same-cell;}-->
+  <span data-sheets-value="{'1':2,'2':'Jakarta XML Web Services'}" data-sheets-userformat="{'2':513,'3':{'1':0},'12':0}">Jakarta XML Web Services</span> - 2.3<br>
```
Comment (bshannon) [inline install/jaxws/docs/assertions/JAXWS23JavadocAssertions.html]: More gratuitous html.

File: install/jaxws/docs/assertions/JAXWS23SpecAssertions.html [hunk 14]
```diff
@@ -24,7 +24,9 @@
 <body bgcolor="white">
 <br>
 <CENTER>
-<h2>Java(TM) API for XML Web Services<br>JAXWS - 2.3<br>
+<h2><br> 
+  <!--td {border: 1px solid #ccc;}br {mso-data-placement:same-cell;}-->
+  <span data-sheets-value="{'1':2,'2':'Jakarta XML Web Services'}" data-sheets-userformat="{'2':513,'3':{'1':0},'12':0}">Jakarta XML Web Services</span> - 2.3<br>
```
Comment (bshannon) [inline install/jaxws/docs/assertions/JAXWS23SpecAssertions.html]: Ditto.

File: install/jaxws/docs/wsimport.html [hunk 15]
```diff
@@ -86,7 +86,9 @@ <h3>Syntax</h3>
       <p><code>-b &lt;path&gt;</code></p>
       </td>
       <td>
-      <p>Specify external JAX-WS or JAXB binding files (Each <code>&lt;file&gt;</code>
+      <p>Specify external Jakarta XML Web Services or 
+        <!--td {border: 1px solid #ccc;}br {mso-data-placement:same-cell;}-->
+        <span data-sheets-value="{'1':2,'2':'Jakarta XML Binding'}" data-sheets-userformat="{'2':513,'3':{'1':0},'12':0}">Jakarta XML Binding</span> (JAXB) binding files (Each <code>&lt;file&gt;</code>
```
Comment (bshannon) [inline install/jaxws/docs/wsimport.html]: Ditto.

File: install/jaxws/docs/wsimport.html [hunk 16]
```diff
@@ -313,7 +319,9 @@ <h3><a name="Ant_task"></a>Ant task<br>
       <p><code>binding</code></p>
       </td>
       <td width="70%">
-      <p>Specify external JAX-WS or JAXB binding files</p>
+      <p>Specify external Jakarta XML Web Services or 
+        <!--td {border: 1px solid #ccc;}br {mso-data-placement:same-cell;}-->
+        <span data-sheets-value="{'1':2,'2':'Jakarta XML Binding'}" data-sheets-userformat="{'2':513,'3':{'1':0},'12':0}">Jakarta XML Binding</span> binding files</p>
```
Comment (bshannon) [inline install/jaxws/docs/wsimport.html]: Ditto x3.

File: install/jms/ra/docs/CTS7-SA-JMSRA-Testing.html [hunk 17]
```diff
@@ -38,85 +38,89 @@
   <body bgcolor="#ffffdd" text="#000000" link="#0000ff" vlink="#840084" alink="#0000ff">
 
 <div align="Center">
-<h1>Setup and configuration of Java EE 7 RI and CTS 7 for testing a standalone JMS Resource Adapter</h1>
+<h1>Setup and configuration of Jakarta EE 8 CI and CTS 8 for testing a standalone<br>
```
Comment (bshannon) [inline install/jms/ra/docs/CTS7-SA-JMSRA-Testing.html]: Weird that this is in a file named "CTS7".

File: install/jpa/docs/index.html [hunk 18]
```diff
@@ -35,25 +35,27 @@
...
-  <li><p>The <a href="assertions/api/api-assertions-2_2.html">Javadoc Assertion List</a> lists all the javadoc assertions that are tested by the JPA TCK.</p></li>
-  <li><p>The <a href="assertions/spec/spec-asssertions-2_2.html">Specification Assertion List</a> lists all the specification assertions that are tested by the JPA TCK.</p></li>
+  <li>The <em>Jakarta Persistence Technology Compatibility Kit, Version 2.2 User's Guide </em> provides the information that you need to install, set up, and  run  the Jakarta Persistence TCK, Version 2.2.  In addition, the guide provides the rules  you must comply with to pass the Jakarta Persistence TCK.</li>
+  <li>The <a href="assertions/api/api-assertions-2_2.html">Javadoc Assertion List</a> lists all the javadoc assertions that are tested by the Jakarta Persistence TCK.</li>
+  <li>he <a href="assertions/spec/spec-asssertions-2_2.html">Specification Assertion List</a> lists all the specification assertions that are tested by the Jakarta Persistence TCK.</li>
```
Comment (bshannon) [inline install/jpa/docs/index.html]: "he" -> "The"

File: install/jsf/docs/ReleaseNotes.html [hunk 19]
```diff
@@ -54,38 +54,38 @@ <h1>JSF Technology Compatibility Kit, Version 2.3.0<br>
...
 
-<p>Refer to the <cite>JSF TCK 2.3.0 User's Guide</cite> for platform compatibility requirements and notes.</p>
+<p>Refer to the <cite>Jakarta Faces TCK 2.3.0 User's Guide</cite> for platform compatibility requirements and notes.</p>
 
 <hr>
-<h2>Reference Implementation</h2>
+<h2>Compatible Implementation</h2>
 
-<p>The reference implementation for this release is Mojarra Java
+<p>A  compatible implementation for this release is Eclipse Mojarra Java
```
Comment (bshannon) [inline install/jsf/docs/ReleaseNotes.html]: Fix "Java ServerFaces".

File: install/jstl/docs/ReleaseNotes-jstl.html [hunk 20]
```diff
@@ -105,34 +104,35 @@
 
 <body>
 <div align="center">
-<h1>A Standard Tag Library for JavaServer Pages<small><sup>TM</sup></small>
-(JSTL) Technology Compatibility Kit, Version 1.2<br>
-<span class="emphasize">Release Notes, March 2010</span></h1>
+<h1><span data-sheets-value="{'1':2,'2':'Jakarta Standard Tag Library'}" data-sheets-userformat="{'2':513,'3':{'1':0},'12':0}">Jakarta Standard Tag Library</span> Technology Compatibility Kit, Version 1.2<br>
+<span class="emphasize">Release Notes, September 2019</span></h1>
```
Comment (bshannon) [inline install/jstl/docs/ReleaseNotes-jstl.html]: More gratuitous html.

File: install/jstl/docs/ReleaseNotes-jstl.html [hunk 21]
```diff
@@ -155,52 +155,36 @@ <h2>Platform Notes</h2>
 <a name="ReferenceRuntime">
```
Comment (bshannon) [inline install/jstl/docs/ReleaseNotes-jstl.html]: Should this all be "Compatible Runtime", as you've changed it elsewhere?

File: install/caj/docs/CAJ1.3-ReleaseNotes.html [hunk 22]
```diff
@@ -38,44 +38,47 @@
...
+<li>MAC OS X, <em>El Capitan</em> (10.9.5+)</li>
+<li>Oracle Linux 7.1</li>
+<li>Windows 10</li>
 </ul>
 
-<p>The CAJ TCK tests have been run against the following Common Annotations for Java  1.3 Reference Implementations:</p>
+<p>The Jakarta Annotations TCK tests have been run against the following Common Annotations for Jakarta EE compatible implementations:</p>
 <ul>
-<li>Oracle Common Annotations for Java Reference Implementation Version 1.3</li>
+<li>Jakarta  Annotations for Java Reference Implementation Version 1.3</li>
```
[the remark this answers is not in the document]
Comment (edbratt) [reply install/caj/docs/CAJ1.3-ReleaseNotes.html]: will fix

File: install/jakartaee/docs/index.html (hunk 6 above)
Comment (edbratt) [reply install/jakartaee/docs/index.html]: will fix

File: install/jaspic/docs/assertions/JASPICJavadocAssertions.html (hunk 7 above)
Comment (edbratt) [reply install/jaspic/docs/assertions/JASPICJavadocAssertions.html]: I guess I didn't use paste text only.

File: install/jaxr/docs/JAXRJavadocAssertions.html (hunk 10 above)
Comment (edbratt) [reply install/jaxr/docs/JAXRJavadocAssertions.html]: Seems to be something to do with the idea that these assertions could span multiple Spec versions. I'd just as soon leave this.

**Pull request merged**